tests/SiteConfigTest.php

```diff
@@ -11,18 +11,18 @@ class SiteConfigTest extends SapphireTest
 {
     protected static $fixture_file = 'SiteConfigTest.yml';
 
-    protected $illegalExtensions = array(
-        'SilverStripe\\CMS\\Model\\SiteTree' => array('SiteTreeSubsites')
+    protected static $illegal_extensions = array(
+        'SilverStripe\\CMS\\Model\\SiteTree' => array('SiteTreeSubsites'),
     );
 
-    public function setUpOnce()
```

Comment:
I believe that @tractorcow has removed the need for this — non-existent classes will be ignored.